src/client/pages/ToeristischeVerhuur/ToeristischeVerhuur.tsx

```diff
@@ -395,7 +395,7 @@ export default function ToeristischeVerhuur() {
         title="Vergunningen"
         hasItems={!!vergunningen.length}
         noItemsMessage={BB_VERGUNNING_DISCLAIMER}
-        startCollapsed={profileType !== 'commercial'}
+        startCollapsed={isCollapsed('vergunningen')}
```

Comment:
Volgens mij is de switch statement voor vergunningen ook incorrect. Ik denk dat het als volgt moet zijn:

```
case 'vergunningen':
        // Start ingeklapt als er geplande, geannuleerde of verlopen verhuur is
        return (
          !!plannedVerhuur.length ||
          !!cancelledVerhuur.length ||
          !!previousVerhuur.length ||
          !!verhuur.length
        );
```